### PR TITLE
[DOC] Documentation for builtin normalize function

### PR DESCRIPTION
**Pull request by escher-m**
I have tried adding documentation for the normalize function. Please review changes.

### CODE REVIEW

File: dev/docs/builtins-reference.md [hunk 1]
```diff
@@ -351,3 +353,32 @@ y = X %*% rand(rows = ncol(X), cols = 1)
 w = lm(X = X, y = y)
 ress = slicefinder(X = X,W = w, Y = y,  k = 5, paq = 1, S = 2);
 ```
+## `normalize`-Function
+
+The `normalize`-function is used for data preparation by normalization. Here, the values of a matrix (numeric columns) are changed in the dataset to use a common scale, while preserving differences in the ranges of values.
```
Comment (j143) [inline dev/docs/builtins-reference.md:358]: The lines can be wrapped for readability. :)

358 .... (numeric columns)
359 are changed ...

File: dev/docs/builtins-reference.md [hunk 2]
```diff
@@ -351,3 +353,32 @@ y = X %*% rand(rows = ncol(X), cols = 1)
...
+| Type           | Description |
+| :------------- | :---------- |
+| Matrix[Double] | 1-column matrix of normalized values. |
+
+
+
+### Example
+```r
+X = rand (rows = 50, cols = 10)
+y = X %*% rand(rows=ncol(X), cols=1)
```
Comment (j143) [inline dev/docs/builtins-reference.md:381]: In the first line: `X = rand(rows = 50, cols = 10)`
In the second line: `y = ..... rand(rows = ncol(X), cols = 1)`

Let's use the same style through out!

File: dev/docs/builtins-reference.md [hunk 3]
```diff
@@ -351,3 +353,32 @@ y = X %*% rand(rows = ncol(X), cols = 1)
...
+y=normalize(X = X)
```
Comment (j143) [inline dev/docs/builtins-reference.md:382]: Also, `y = normalize(X = X)` or `y=normalize(X = X)` can be used consistently.

**Comment by escher-m** [comment]
Made the code consistent

**Comment by j143** [comment]
Thank you, @escher-m - LGTM. 👍 
 🥳 🚀

**Pull request closed without merging**